tests/test_memory.py

```diff
@@ -65,16 +65,19 @@ def __init__(self):
                          serverPort=None)
 
         # Use a memory space emulator
-        self.sim = pr.interfaces.simulation.MemEmulate()
+        sim = pr.interfaces.simulation.MemEmulate()
+        self.addInterface(sim)
```

Comment:
Are future rogue scripts going to have to use `self.addInterface()` to add interfaces to the py.root or will the standard method that we use everywhere still work?

Reply:
addInterface() is only really a helper. existing scripts will not break.